v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -0,0 +1,272 @@
+const get = require("get-value");
+const { logger } = require("handlebars");
+const { isArray } = require("lodash");
+const sha256 = require("sha256");
+const {
+  isDefinedAndNotNullAndNotEmpty,
+  returnArrayOfSubarrays,
+  CustomError,
+  constructPayload,
+  defaultRequestConfig,
+  getSuccessRespEvents,
+  getErrorRespEvents,
+  removeUndefinedAndNullAndEmptyValues
+} = require("../../util");
+const {
+  offlineDataJobsMapping,
+  addressInfoMapping,
+  BASE_ENDPOINT,
+  attributeMapping,
+  hashAttributes
+} = require("./config");
+
+const hashEncrypt = object => {
+  Object.keys(object).forEach(key => {
+    if (hashAttributes.includes(key)) {
+      // eslint-disable-next-line no-param-reassign
+      object[`${key}`] = sha256(object[`${key}`]);
```

Comment:
the keys in `hashAttributes` are all string values only right? what is the purpose of `${key}` here?